reconciletags/config_integrity_test.py

```diff
@@ -0,0 +1,51 @@
+"""Tests to check the integrity of json config files.
+
+These tests assume that the json configs live in a top-level
+folder named config."""
+
+import glob
+import json
+import subprocess
+import unittest
+
+
+class ReconcilePresubmitTest(unittest.TestCase):
+
+    def _get_digests(self, repo):
+        output = json.loads(
+            subprocess.check_output(['gcloud', 'beta', 'container',
```

Comment:
How will these get run? Could you add a comment that mentions gcloud must be installed and authenticated?